include/openmc/constants.h

```diff
@@ -318,7 +318,9 @@ enum class Interpolation {
   lin_lin = 2,
   lin_log = 3,
   log_lin = 4,
-  log_log = 5
+  log_log = 5,
+  quadratic = 6,
```

Comment:
The ENDF-6 format actually uses 6 for "special one-dimensional interpolation law, used for charged-particle cross sections only", so we might want to avoid this value and use 7 and 8 for quadratic and cubic instead.